.gitattributes

```diff
@@ -0,0 +1,7 @@
+data/kaggle-ground-truth/*.jpg filter=lfs diff=lfs merge=lfs -text
+data/kaggle-ground-truth/*.csv filter=lfs diff=lfs merge=lfs -text
+*.box filter=lfs diff=lfs merge=lfs -text
```

Comment:
we don't want to be tracking `.box` or `.lstmf` files as these are artifacts generated by tesseract, not training data